### PR TITLE
feat: Added spending plans for testing on testnet.

### LINKED ISSUE #3111: The test framework needs an option to load test accounts from the spending plan in order to run on testnet and mainnet.
### Description

Currently, the only static account used in the test framework is the relay operator's account.   Accounts are created in the framework by creating alias accounts and generating keys while the tests are running.  This makes it impossible to whitelist or add these accounts to spending plans that will not trigger HBar Rate Limiter errors. 
 
When an HBar Rate Limiter error is triggered, the framework does not refund the HBars spent.  This is the cause of the mainnet test "leak" of funds.   

This is a real issue when running tests on testnet or mainnet.

### Steps to reproduce

Run some tests that can trigger the HBar Rate Limiter on testnet or mainnet.

### Additional context

_No response_

### Hedera network

mainnet

### Version

v0.57.2

### Operating system

Linux

### PR DESCRIPTION
**Pull request by ebadiere**
feat: Moved create aliases script to the sctrips directory.

**Description**:
This PR replaces PR #3116 with minimal package.json changes.

**Related issue(s)**:

Fixes #3111 

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: scripts/createAliasAccounts.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    const aliasAccountId = AccountId.fromEvmAddress(0, 0, ethereumAddress);
+
+    const transferTx = await new TransferTransaction()
+      .addHbarTransfer(operatorAccountId, initialBalanceTinybar.negated())
+      .addHbarTransfer(aliasAccountId, initialBalanceTinybar)
+      .execute(client);
+
+    const receipt = await transferTx.getReceipt(client);
+    console.log(`Transfer Transaction Status: ${receipt.status.toString()}`);
```
Comment (quiet-node) [inline scripts/createAliasAccounts.js:68]: This works fine, but I've seen the `AccountCreateTransaction` API used before, like this:

```
TransactionResponse newAccount = new AccountCreateTransaction()
     .setKey(newAccountPublicKey)
     .setInitialBalance(Hbar.fromTinybars(1000))
     .execute(client);
```

Just a suggestion—no need to change anything!

**Review by quiet-node — APPROVED** [review]
LGTM

**Review by natanasow — APPROVED** [review]
LGTM

**Comment by ebadiere** [comment]
Covered by #3201

**Pull request closed without merging**